test/BlockManager.js

```diff
@@ -122,6 +125,17 @@ describe('BlockManager', function () {
 
       await voteManager.connect(signers[6]).commit(epoch, commitment2);
 
+      const votes3 = [100, 200, 300, 400, 500, 600, 700, 800, 900];
```

Comment:
Why do we need for your new staker : 8 to participate here ?

Reply:
So for testing this particular edge testCase I decided to add the new staker which will only participate in commit and reveal state and I have commited and revealed it two times  before testing it's stake in 'no block should be confirmed in previous epoch' test case .

Reply:
Yep adding new staker is fine, but why we need for them to commit and reveal here, doesnt it make this testcase needlessly bulky ?

Reply:
Yes